.vscode/settings.json

```diff
@@ -1,2 +0,0 @@
-{
```

Comment:
**suggestion:** Add  `.vscode` to .gitignore file

There is no need to add your VSCode settings to the repository

Reply:
Did it!

Reply:
Also on that kind of comments after I said I did it and responded to you I should resolve it or not because I don't want to be inpolite but it's nothing to discuss I thing because it's just something that needs to be done?